src/TestEnvironment.Docker/Container.cs

```diff
@@ -181,7 +205,15 @@ async Task<ContainerListResponse> CreateContainer()
                 if (Ports != null)
                 {
                     createParams.HostConfig.PortBindings = Ports
-                        .ToDictionary(p => $"{p.Key}/tcp", p => (IList<PortBinding>)new List<PortBinding> { new PortBinding { HostPort = p.Value.ToString() } });
+                        .ToDictionary(
+                            p => $"{p.Key}/tcp",
+                            p => (IList<PortBinding>)new List<PortBinding>
+                                { new PortBinding { HostPort = p.Value.ToString() } });
+                }
+
+                if (Entrypoint != null && Entrypoint.Any())
+                {
+                    createParams.Entrypoint = Entrypoint;
```

Comment:
So, actually, your merge request contains additional configuration value for specifying EntryPoint? Please fill description with summary of changes in your pull request

Reply:
The description has been updated.